tools/hpo/configs/quality_score_hpo.yaml

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
The format of HPO config or how to write a HPO config should be introduced in HPO README files.